s3parq/publish_redshift.py

```diff
@@ -0,0 +1,78 @@
+from s3parq.session_helper import SessionHelper
+from sqlalchemy import Column, Integer, String
+import logging, re
+import pandas as pd
+
+logger = logging.getLogger(__name__)
+
+def _check_reserved_keyword(name: str):
+    reserved = "AES128, AES256, ALL, ALLOWOVERWRITE, ANALYSE, ANALYZE, AND, ANY, ARRAY, AS, ASC, AUTHORIZATION, BACKUP, BETWEEN, BINARY, BLANKSASNULL, BOTH, BYTEDICT, BZIP2, CASE, CAST, CHECK, COLLATE, COLUMN, CONSTRAINT, CREATE, CREDENTIALS, CROSS, CURRENT_DATE, CURRENT_TIME, CURRENT_TIMESTAMP, CURRENT_USER, CURRENT_USER_ID, DEFAULT, DEFERRABLE, DEFLATE, DEFRAG, DELTA, DELTA32K, DESC, DISABLE, DISTINCT, DO, ELSE, EMPTYASNULL, ENABLE, ENCODE, ENCRYPT, ENCRYPTION, END, EXCEPT, EXPLICIT, FALSE, FOR, FOREIGN, FREEZE, FROM, FULL, GLOBALDICT256, GLOBALDICT64K, GRANT, GROUP, GZIP, HAVING, IDENTITY, IGNORE, ILIKE, IN, INITIALLY, INNER, INTERSECT, INTO, IS, ISNULL, JOIN, LANGUAGE, LEADING, LEFT, LIKE, LIMIT, LOCALTIME, LOCALTIMESTAMP, LUN, LUNS, LZO, LZOP, MINUS, MOSTLY13, MOSTLY32, MOSTLY8, NATURAL, NEW, NOT, NOTNULL, NULL, NULLS, OFF, OFFLINE, OFFSET, OID, OLD, ON, ONLY, OPEN, OR, ORDER, OUTER, OVERLAPS, PARALLEL, PARTITION, PERCENT, PERMISSIONS, PLACING, PRIMARY, RAW, READRATIO, RECOVER, REFERENCES, RESPECT, REJECTLOG, RESORT, RESTORE, RIGHT, SELECT, SESSION_USER, SIMILAR, SNAPSHOT , SOME, SYSDATE, SYSTEM, TABLE, TAG, TDES, TEXT255, TEXT32K, THEN, TIMESTAMP, TO, TOP, TRAILING, TRUE, TRUNCATECOLUMNS, UNION, UNIQUE, USER, USING, VERBOSE, WALLET, WHEN, WHERE, WITH, WITHOUT".split()
```

Comment:
I noticed we do with above as well.  The algorithm seems to be:
1) remove the whitespace from this hard-coded string and split into a list
2) remove the commas from each string in the list
3) check if `name.upper` is in the list

Since we're hardcoding this, why don't we define `reserved` exactly how we want it, and we can skip the `.split` and `.strip(',')` stuff?  It would be 
```
reserved = ['AE128', 'AE256', etc...]
return (name.upper() in reserved)
```

We also do this in `publish_parq.py`, so we should probably change it there as well.